modules/datadict/php/datadict.class.inc

```diff
@@ -60,11 +60,20 @@ class Datadict extends \DataFrameworkMenu
     {
         $instruments     = \Utility::getAllInstruments();
         $dictInstruments = [];
+        $lorisinstance   = new \LORIS\LorisInstance(
```

Comment:
There is no reason to instantiate a new object here, it should just use $this->lorisinstance

Reply:
fixed